tests/testthat.R

```diff
@@ -22,6 +22,18 @@ db_spctra_file <- createCompDb(cmps, metadata = metad2, path = tempdir(),
                                msms_spectra = msms_spctra)
 cmp_spctra_db <- CompDb(db_spctra_file)
 
+# why are there duplicated rows in the ms_compound table with the same compound_id?
```

Comment:
I guess that's because the same compound was measured several time - but not so sure. Problem is we can not control the data we get as input from e.g. MassBank or HMDB. it's certainly not the nicest, cleanest data...